### DISCUSSION TITLE
Make switching between accounts faster

### DISCUSSION THREAD

**Discussion by mvilanova** [community/community#130800] [Mobile]
Right now, switching between accounts requires too many steps in my opinion. It would be great if you could click on the account on the bottom left and switch to another one.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.167.0 (134878402)
Device: iPad16,4 (iOS 17.5.1)
</details>

**Comment by alcere** [comment]
Hi @mvilanova! Thanks for the feedback. I want to make sure I got your suggestion right. You want to switch accounts by tapping the account avatar on the Profile tab, right? You might find it useful to know that you can also show the account switcher by holding down the Profile tab.

**Reply by mvilanova** [reply]
Ooohhh I didn't know I could do that! That's perfect. Thanks!
